.gitignore

```diff
@@ -22,3 +22,6 @@ third_party/lzma.js/lzma-native.exe
 tools/optimizer_build/
 
 .DS_Store
+
+.coverage*
```

Comment:
What are these dotfiles for?

Reply:
These are temporary files generated by the python `coverage` tool. The tool likes to pollute the current working directory.